website/content/api-docs/system/auth.mdx

```diff
@@ -188,8 +188,11 @@ $ curl \
 
 ```json
 {
-  "default_lease_ttl": 3600,
-  "max_lease_ttl": 7200
+  "default_lease_ttl": 2764800,
```

Comment:
I'm actually getting a response like this, when I try to hit this endpoint, should we add these other fields?
```
{
    "description": "",
    "default_lease_ttl": 2764800,
    "max_lease_ttl": 2764800,
    "force_no_cache": false,
    "token_type": "default-service",
    "request_id": "b814023c-4f0f-cf19-6ef2-fe666037c1bf",
    "lease_id": "",
    "renewable": false,
    "lease_duration": 0,
    "data": {
        "default_lease_ttl": 2764800,
        "description": "",
        "force_no_cache": false,
        "max_lease_ttl": 2764800,
        "token_type": "default-service"
    },
    "wrap_info": null,
    "warnings": null,
    "auth": null
}
```

Reply:
For the consistency, we should keep it looks like all other API docs.  It seems that the API docs only shows the HTTP response payload as the sample response.

Reply:
ah yep, that is true, I did notice this isn't the only endpoint with this difference, so that sounds good to me!